app/main.go

```diff
@@ -27,7 +27,7 @@ func main() {
 
 	port := ":8080"
```

Comment:
remove port variable (move it to your config json) that use it as port variable has been used (in ListenAndServe func and in fmt.Println*(Server started at....)

Reply:
you mean ctrl+x ctrl+v or what? :D I can't understand. I had googled it but i didn't find answer.

Reply:
you just need to remove port variable, than go outliner will tell you where you need to set Config.ListenPort instead of port var (it will underline places).

Reply:
DONE!